<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostFrame</class>
 <widget class="QFrame" name="PostFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QToolButton" name="arrowButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>13</y>
     <width>11</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/res/arrow.gif);
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #3f3f3f;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #3f3f3f;</string>
   </property>
   <property name="text">
    <string>00/00/00</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #797979;</string>
   </property>
   <property name="text">
    <string>de Fernando Jim√©nez</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>4</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #6699ff;
border: 0px;
}
QPushButton:hover {
color: #ff6600;
}
</string>
   </property>
   <property name="text">
    <string>Suprimir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>4</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #6699ff;
border: 0px;
}
QPushButton:hover {
color: #ff6600;
}
</string>
   </property>
   <property name="text">
    <string>Visualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>4</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #6699ff;
border: 0px;
}
QPushButton:hover {
color: #ff6600;
}
</string>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="contentTextEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
color: #3f3f3f;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
